types/node/globals.d.ts

```diff
@@ -1,5 +1,28 @@
+// Error extensions
+interface Error {
+    /**
+     * If present, the `error.cause` property is the underlying cause of the `Error`.
+     * It is used when catching an error and throwing a new one with a different message or code in order to still have access to the original error.
+     * @added v16.9.0
+     */
+    cause?: Error | undefined;
+}
+
+/**
+ * @added v16.9.0
+ */
+interface ErrorOptions {
+    /**
+     * The error that caused the newly created error.
+     * @added v16.9.0
+     */
+    cause?: Error["cause"];
+}
+
 // Declare "static" methods in Error
 interface ErrorConstructor {
+    new(message?: string, options?: ErrorOptions): Error;
+    (message?: string, options?: ErrorOptions): Error;
```

Comment:
I'm unsure if we should expose the non `new` version, this would hide the fact that this is actually a constructor call, is there any upside to including it?

Reply:
I'm always of the opinion that not everything that works at runtime should be encouraged but that's just me, feel free to go head w/ this.